src/Tests/ComponentTests/Sagas/SagaWithoutCorrelationPropertyData.cs

```diff
@@ -5,7 +5,7 @@
     using Extensibility;
     using Sagas;
 
-    public class SagaWithoutCorrelationProperty : Saga<SagaWithoutCorrelationPropertyData>, 
+    public class SagaWithoutCorrelationProperty : Saga<SagaWithoutCorrelationPropertyData>,
```

Comment:
it looks like the related test is requiring finder support which we have disabled in the actual persistence tests anyway? So that doesn't seem to cover a scenario that needs to be covered?

Reply:
I looked into it just to make sure we weren't missing anything. This test doesn't actually require finders support at all. It's passing in the finder's type and down the line, it is not used. SF indeed does not support finders, so I have fixed that as well so we don't see any conflicts across the configurations within the solution.